npc/re/instances/IsleOfBios.txt

```diff
@@ -0,0 +1,764 @@
+	'vrid3$ = instance_npcname("Vrid#bios3");
+	initnpctimer;
+	npctalk(_("You're lucky to be here. I got totally lost the moment I entered that maze."), 'vrid3$);
+	end;
+OnTimer2500:
+	npctalk(_("If it weren't for Zeith, I'd still be in there, wandering. Thanks, Zeith."), 'vrid3$);
+	end;
+OnTimer5000:
+	npctalk(_("...Something's been bothering me."), 'zeith3$);
+	end;
+OnTimer7500:
+	npctalk(_("What's up?"), 'vrid3$);
+	end;
+OnTimer10000:
+	npctalk(_("I've got the feeling that the monsters were growing stronger with time."), 'zeith3$);
+	end;
+OnTimer12500:
+	npctalk(_("Just like children growing up into adults."), 'zeith3$);
+	end;
+OnTimer15000:
+	npctalk(_("Um, I think you're right."), 'vrid3$);
+	end;
+OnTimer17500:
+	npctalk(_("They're all small in the beginning, and then we faced bigger and stronger ones over time."), 'vrid3$);
+	end;
+OnTimer20000:
+	npctalk(_("To me, that sounds natural. What are you worried about?"), 'vrid3$);
+	end;
+OnTimer22500:
+	npctalk(_("Well, I'm curious about the monster we'll face now."), 'zeith3$);
+	end;
+OnTimer25000:
+	npctalk(_("Whether they're stronger versions of their original selves, or something completely different."), 'zeith3$);
+	end;
+OnTimer27000:
+	enablenpc 'reaper3$;
+	end;
+OnTimer27500:
+	npctalk(_("*Chuckle* At least one of you has a good head on his shoulders."), 'reaper3$);
+	end;
+OnTimer30000:
+	npctalk(_("You again?"), 'vrid3$);
+	end;
+OnTimer32500:
+	npctalk(_("Are you curious about your next adversaries?"), 'reaper3$);
+	end;
+OnTimer35000:
+	npctalk(_("Ahead of you is an army of the immortal."), 'reaper3$);
+	end;
+OnTimer37500:
+	npctalk(_("*Chuckle* Come out, my soldiers!"), 'reaper3$);
+	end;
+OnTimer37700:
+	specialeffect EF_READYPORTAL,AREA,'reaper3$;
+	end;
+OnTimer40000:
+	for ( .@i = 1; .@i <= 3; ++.@i ) {
+		for ( .@j = 1; .@j <= 8; ++.@j )
+			enablenpc instance_npcname( "Corrupt Orc Zombie#" + .@j + "bio" + .@i );
+	}
+	end;
+OnTimer42000:
+	npctalk(_("Shut up, you fiend! We don't have time for your lackeys!"), 'vrid3$);
+	end;
+OnTimer45000:
+	npctalk(_("I'll get rid of you myself!"), 'vrid3$);
+	end;
+OnTimer47500:
+	npctalk(_("Right back at you, bigmouth."), 'reaper3$);
+	end;
+OnTimer50000:
+	npctalk(_("I'll teach you how terrifying it is to be helpless in the face of horror!"), 'reaper3$);
+	end;
+OnTimer52500:
+	specialeffect EF_STORMGUST,AREA,'reaper3$;
+	end;
+OnTimer52700:
+	specialeffect EF_FREEZED,AREA,'zeith3$;
+	specialeffect EF_FREEZED,AREA,'vrid3$;
+	specialeffect EF_FIREHIT,AREA,'zeith3$;
+	specialeffect EF_FIREHIT,AREA,'vrid3$;
+	setnpcdisplay( 'vrid3$, 4_M_FROZEN_KN );
+	setnpcdisplay( 'zeith3$, 4_M_FROZEN_GC );
+	getpartymember(getcharid(CHAR_ID_PARTY), PT_MEMBER_NAME, .@player_name$);
+	getpartymember(getcharid(CHAR_ID_PARTY), PT_MEMBER_CHARID, .@char_id );
+	getpartymember(getcharid(CHAR_ID_PARTY), PT_MEMBER_ACCID, .@account_id );
+	.@count = getpartymember(getcharid(CHAR_ID_PARTY), PT_MEMBER_ACCID, .@account_id);
```

Comment:
a bit of optmization, but can't we save a call here by simply getting count from line 477 instead of calling it twice?